mlir/lib/Bindings/Python/Rewrite.cpp

```diff
@@ -425,9 +427,9 @@ void mlir::python::populateRewriteSubmodule(nb::module_ &m) {
           "results.")
       .def(
           "apply_patterns_and_fold_greedily",
-          [](PyOperationBase &op, MlirFrozenRewritePatternSet set) {
+          [](PyOperationBase &op, PyFrozenRewritePatternSet &set) {
```

Comment:
It seems this definition is duplicated with line 414 (above)? Maybe we can just remove it?